addons/payment_razorpay/models/payment_transaction.py

```diff
@@ -168,6 +170,21 @@ def _razorpay_get_mandate_max_amount(self):
             max_amount = pm_max_amount
         return max_amount
 
+    @api.model
+    def _razorpay_convert_inr_to_tx_currency(self, amount, currency_id):
+        """ Convert INR amount to given currency.
+
+        :param float amount: The amount to be convert(in INR).
+        :param currency_id: The currency amount should be converted.
+        :return: The converted amount in given currency.
+        :rtype: float
+        """
+        INR_currency = self.env['res.currency'].with_context(active_test=False).search([
+            ('name', '=', 'INR'),
+        ], limit=1)
+        converted_amount = INR_currency._convert(amount, currency_id)
+        return float_round(converted_amount, precision_digits=0)
```

Comment:
I updated this to convert based on INR's rounding; please adapt if it's not correct.

Reply:
Normally all the currency have 6 digits after .(point) and it was not looking good for just message that's why I need use `float_round`

Reply:
Ok but the method is used in another place already and probably more in the future. Maybe you could round specifically for the message in the get_warning method?